src/main/scala/rocket/RVC.scala

```diff
@@ -16,7 +16,7 @@ class ExpandedInstruction extends Bundle {
   val rs3 = UInt(5.W)
 }
 
-class RVCDecoder(x: UInt, xLen: Int, useAddiForMv: Boolean = false) {
+class RVCDecoder(x: UInt, xLen: Int, useAddiForMv: Boolean = false, usingBitManip: Boolean = false, usingMulDiv: Boolean = false, usingCompressedSuiteB: Boolean = false) {
```

Comment:
Can you add some comments describing how ISA features map to these flags

Reply:
Following the current convention, I assume we should use

```
class RVCDecoder(x: UInt, xLen: Int) extends HasCoreParameters {
```
then we can use these `using`s freely.

But as [chipsalliance/rocket](https://github.com/chipsalliance/rocket) is trying to remove `HasCoreParameters` everywhere and put `using`s to the class parameter, I do not know which one is recommended.

Reply:
I actually tried exactly this in the first place but got the following error: